src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,120 @@
+<%--
+  Copyright 2018 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<%@ page import="java.time.Instant" %>
+<%@ page import="java.util.Date" %>
+<%@ page import="java.util.Calendar" %>
+<%@ page import="java.util.TimeZone" %>
+<%@ page import="java.text.DateFormat" %>
+<%@ page import="java.text.SimpleDateFormat" %>
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.Map" %>
+<%@ page import="java.util.UUID" %>
+<%@ page import="codeu.controller.ActivityFeedServlet" %>
+<%@ page import="java.util.PriorityQueue" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+<%@ page import="codeu.model.store.basic.ConversationStore" %>
+<%
+PriorityQueue<ActivityFeedServlet.Item> FeedItem = (PriorityQueue<ActivityFeedServlet.Item>) request.getAttribute("FeedItems");
+Map<UUID, Object> ids = (Map<UUID, Object>) request.getAttribute("ids");
+//List<User> users = (List<User>) request.getAttribute("users");
```

Comment:
nit: remove commented out code. If it doesn't work, then you don't need it. If you need it, it won't be commented out

Reply:
Done